pom.xml

```diff
@@ -49,13 +49,8 @@
       </dependency>
       <dependency>
         <groupId>com.hazelcast</groupId>
-        <artifactId>hazelcast-aws</artifactId>
-        <version>3.4</version>  <!-- https://github.com/hazelcast/hazelcast-aws#requirements -->
-      </dependency>
-      <dependency>
-        <groupId>com.hazelcast</groupId>
-        <artifactId>hazelcast-kubernetes</artifactId>
-        <version>2.2.3</version>  <!-- https://github.com/hazelcast/hazelcast-kubernetes#requirements-and-recommendations -->
+        <artifactId>hazelcast</artifactId>
+        <version>5.2.3</version>
```

Comment:
> Can we completely remove this dependency and simply use the version that vertx-hazelcast ships with?
> 
> https://github.com/vert-x3/vertx-hazelcast/blob/4.4.1/pom.xml#L57-L61

We can not because it will pull in `hazelcast 4.2.7`. We need `hazelcast 5.2.2` and beyond.